testing/integration/build.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021-2023 Protocol Labs
```

Comment:
I _think_ we need to put all of this in a separate `integration-test-actors` create, and depend on it as a `dev-dependency`. Otherwise, we won't be able to publish this framework as a crate (we have some external projects using it).

Reply:
Is that possible? According to the [rust docs](https://doc.rust-lang.org/cargo/reference/build-scripts.html#build-dependencies), the build script that is generated from the `build.rs` file does not have any access to the dependencies in neither `[dependencies]` or `[dev-dependencies]` sections.

Reply:
"It's complicated".

Basically, we _can_ access dependencies in the same workspace, locally. Which is sufficient for testing.

As a matter of fact, we're already relying on this fact. I.e., this build file is building the dependencies.

So what we need to do is move [integration/tests/actor_wasm_path/mod.rs](https://github.com/filecoin-project/ref-fvm/pull/1698/files#diff-7f25eb13ddeb590e16036999191c5a868726128f8114c6b0eb62d273694fab0c) and this file into a separate "test actor bundle" crate.

Reply:
I created a `fvm_test_wasm_builder` crate located in `testing/wasm_builder` and added that as `[dev-dependencies]` in `testing/integration/Cargo.toml`. I was not possible to add `[dev-dependencies]` to `testing/calibration/Cargo.toml` since it contains binaries and not libraries (at least I could not get it working). To address that, `fvm_test_wasm_builder` now builds all wasm test actors (for both integration + calibration).